README.md

```diff
@@ -1080,6 +1080,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Yamini Kaushal](https://github.com/Yamini09-code)
   - [Yamini Singh](https://github.com/yamini-singh169)
   - [Yanuka Deneth](https://github.com/yanukadeneth99)
+  - [Yash Gaur](https://github.com/THEYASHGAUR)
```

Comment:
Please add your name in line 1084, after Yash Choudhary.
Let us know if you need any help doing that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable